packages/trader/package.json

```diff
@@ -86,7 +86,7 @@
     "dependencies": {
         "@deriv/components": "^1.0.0",
         "@deriv/deriv-api": "^1.0.11",
-        "@deriv/deriv-charts": "1.3.5",
+        "@deriv/deriv-charts": "1.4.0",
```

Comment:
```suggestion
        "@deriv/deriv-charts": "^1.4.0",
```

Reply:
@akmal-deriv we don't want to automatically install the latest version, we want an exact one instead. :)